tests require a running mongodb service, which alpine had to drop due to it's license change:
https://gitlab.alpinelinux.org/alpine/aports/-/commit/8a901de31fa055ed591d487e12f8bb9ffcc0df21
--- a/tests/test_mongodb_cache.py
+++ b/tests/test_mongodb_cache.py
@@ -26,5 +26,6 @@
         request.cls.cache_factory = _factory
 
 
+@pytest.mark.skip(reason="alpine had to drop mongo due to it's license change")
 class TestMongoDbCache(CommonTests, ClearTests, HasTests):
     pass
